File: applications/summersim18/wrapped/cpp/sub-showcase/left-sub.cpp
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
#include "../../../../../cpp/numb/numb.hpp"

int main()
{
    Numb::setup("sub-cache1", "", 20, "ignore-cache");
    Numb a(1);
    Numb b(1);
    Numb c(0.001);

    Numb d = (a - b) - c;
    cout << "(1 - 1) - 0.001 = " << d << endl;
}

File: applications/summersim18/wrapped/python/sub-showcase/left-sub.py
from numb import *

if __name__ == '__main__':
    Numb.setup(cache_out='sub-cache1', precision=20, strategy='ignore-cache')
    a = Numb(1)
    b = Numb(1)
    c = Numb(0.001)

    d = (a - b) - c
    print "(1 - 1) - 0.001 = {0}".format(d)
